tests/DCPS/Compiler/keywords/classic/keywords_classic.mpc

```diff
@@ -0,0 +1,15 @@
+project: dcps_test {
+  exename = keywords-classic
+  includes += .
```

Comment:
Is this needed?

Reply:
Yes, [it's used in Multitopic](https://github.com/objectcomputing/OpenDDS/blob/13bc1f9bade4d4b8f9d78ee7eb4311c24867bcdf/tests/DCPS/MultiTopic/classic/MultiTopicTest_Classic.mpc#L11), which is the dual mapping archetype. It adds the generated headers to the include path for `main.cpp` because they're in different directories.